### ISSUE TITLE
Send_at is not sent correctly to the API

### ISSUE THREAD

**Issue by Rolorob** [shawnmclean/Mandrill-dotnet#160]
Send_at parameter is currently sent in the incorrect format.
I captured this with fiddler:

,"send_at":"2015-11-03T18:00:00Z","

It should be in the format YYYY-MM-DD HH:MM:SS.

I'll try if I can propose a fix for this because I need it soon :)

**Comment by Rolorob** [comment]
This is probably broken after you switched to HttpClient. I found the following line in your code:

public static string DATE_TIME_FORMAT_STRING = "yyyy-MM-dd HH:mm:ss";

But it is not referenced anywhere anymore :)

**Comment by Rolorob** [comment]
hmmm, I changed it so it would send in the correct format and it still crashed. So I checked the response (which is not parsed correctly), it said:

{"status":"error","code":10,"name":"PaymentRequired","message":"Email scheduling is only available for accounts with a positive balance."}

So I reverted my changes, and I saw that I got the same message. Not sure how important it is to change the format, I will make sure to get some money on my account and then test again.
